packages/millicast-sdk/src/View.js

```diff
@@ -196,6 +202,7 @@ export default class View extends BaseWebRTC {
       subscriberData = await this.tokenGenerator()
       //  Set the iceServers from the subscribe data into the peerConfig
       this.options.peerConfig.iceServers = subscriberData?.iceServers
+      this.options.peerConfig.encodedInsertableStreams = true
```

Comment:
I know we have a try/catch around it, but should we check if this is supported by the browser?
https://github.com/w3c/webrtc-encoded-transform/blob/main/explainer.md
```suggestion
      this.options.peerConfig.encodedInsertableStreams = window.RTCRtpSender && "transform" in RTCRtpSender.prototype;
```

Reply:
Will add this, without the `this` as it is in imported variable